src/main/java/com/zendesk/maxwell/MaxwellTableCache.java

```diff
@@ -22,7 +23,7 @@ public void processEvent(Schema schema, MaxwellFilter filter, TableMapEvent even
 
 			Table tbl = db.findTable(tblName);
 
-			if ( filter != null && filter.isTableBlacklisted(tblName) )
+			if ( filter != null && filter.isTableBlacklisted(tbl.getDatabase().getName(), tblName) )
```

Comment:
I'm developing in windows, and I get errors when running you're test suite... but i did just notice that just above that line there is already a `dbName` variable in scope (on line 17).

So I've switched to just using that.